models.ts

```diff
@@ -14,10 +14,11 @@ export class State {
 export class District {
 
     public name: string;
+    public type: string;
```

Comment:
`type` sounds a bit too generic. Let's use `county` instead.

Reply:
Makes sense, given the other change